pkgs/applications/science/programming/scyther/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, callPackage, callPackage_i686, buildEnv
+, includeGUI ? true
```

Comment:
The package is advertised as a GUI application first, it provides visualizations the cli can't do easily.